app/src/main/res/layout/fragment_new_pin.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true"
+    android:id="@+id/newpin_scrollview">
```

Comment:
I think you have to put the scroll view within the constraint layout so it doesn't shrink and pull the navbar up with it.